<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetDownloadDEM</class>
 <widget class="QWidget" name="WidgetDownloadDEM">
  <property name="windowTitle">
   <string>Download elevation file</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../wn-resources.qrc">
    <normaloff>:/wn-icon.png</normaloff>:/wn-icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWebView" name="wvGoogleMaps">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
       <property name="renderHints">
        <set>QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblDEMSource">
       <property name="text">
        <string>Data Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbDEMSource">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="maxVisibleItems">
        <number>10</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <item>
        <property name="text">
         <string extracomment="US coverage Shuttle Radar Topography Mission data at 30 meter resolution.  Any existing holes in the data have been filled.">US SRTM (30m)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="World coverage Shuttle Radar Topography Mission data at 90 meter resolution.  Any existing holes in the data have been filled.">WORLD SRTM (90m)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="World coverage Global Multi-resolution Terrain Elevation Data 2010 at 250 meter resolution.">WORLD GMTED (250m)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="FARSITE Landscape File (.lcp) data at 30 meter resolution.">Landscape File</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnDownloadDEM">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Download the elevation file.</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Download File</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/server_go.png</normalon>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Map data © &lt;a href=&quot;https://www.openstreetmap.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;OpenStreetMap &lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://creativecommons.org/licenses/by-sa/2.0/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;CC-BY-SA&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://www.openstreetmap.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;, Imagery © &lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://www.mapbox.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Mapbox&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://www.openstreetmap.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../wn-resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
